Framework/PythonInterface/mantid/fitfunctions.py

```diff
@@ -109,6 +109,50 @@ def __mul__ (self, other):
       if prod.pureMultiplication:
          prod = prod.flatten()
       return prod
+      
+  def __call__(self, x, *params):
+      """ Implement function evaluation, such that
+          func(args) is equivalent func.__call__(args)
+           
+          :param x: x value or list of x values
+          :param *params list of parameter values
+      """
+      from mantid.simpleapi import CreateWorkspace, EvaluateFunction
```

Comment:
There is no need for these imports any more. I forgot to delete the line after my changes.